packages/build/src/core/feature_flags.ts

```diff
@@ -17,6 +17,7 @@ const getFeatureFlag = function (name: string): FeatureFlags {
 export const DEFAULT_FEATURE_FLAGS: FeatureFlags = {
   buildbot_zisi_trace_nft: false,
   buildbot_zisi_esbuild_parser: false,
+  buildbot_zisi_system_log: false,
```

Comment:
I guess this is only for the CLI?

Reply:
No, it's for both CLI-based and CI-based deployments. Why do you ask?

Reply:
For better understanding, these are the defaults that get overridden on CI with the actual ones from devcycle.

Reply:
So you're saying that we don't need to add a flag to this list in order to use it? I tend to add flags here whenever they're meant to be consumed by Netlify Build itself, but I'm happy to stop doing that.

Reply:
This is needed because the CLI consumes netlify/build as well and does not get the flags passed down from dev cycle, so this is the way how it works on the CLI. 

Inside our CI environment, we are passing the flags as comma separated list down to it and gets merged with this list.